docs/en/install-upgrade/serverless-changelog.asciidoc

```diff
@@ -10,6 +10,52 @@ For serverless API changes, refer to https://www.elastic.co/docs/api/changes[API
 
 For serverless changes in Cloud Console, refer to https://www.elastic.co/guide/en/cloud/current/ec-release-notes.html[Elasticsearch Service Documentation: Release notes].
 
+[discrete]
+[[serverless-changelog-03102025]]
+=== March 10, 2025
+
+[discrete]
+[[features-enhancements-03102025]]
+==== New features and enhancements
+* The Create Rule flyout, used by solutions, now features the improved rule form in Elastic Observability Serverless  ({kibana-pull}206685[#206685]).
+* Resolves duplicate conversations in Elastic Observability Serverless ({kibana-pull}208044[#208044]).
+* Split the SLO Details view from the Overview page in Elastic Observability Serverless ({kibana-pull}212826[#212826]).
+* Adds the reason message to the rules recovery context in Elastic Observability Serverless ({kibana-pull}211411[#211411]).
+* Runtime metrics dashboards now support different ingest paths in Elastic Observability Serverless ({kibana-pull}211822[#211822]).
+* Adds SSL options for Fleet Server hosts settings in Fleet ({kibana-pull}208091[#208091]).
+* Introduces globe projection for Dashboards and visualizations ({kibana-pull}212437[#212437]).
+* Registered a custom integrations search provider in Fleet ({kibana-pull}213013[#213013]).
+* Adds support for searchAfter and PIT (point-in-time) parameters in the Get Agents List API in Fleet ({kibana-pull}213486[#213486]).
```

Comment:
Hi @KOTungseth can we add this one from Security? Received the request from the Rules team as an ad-hoc. 
```suggestion
* Adds support for searchAfter and PIT (point-in-time) parameters in the Get Agents List API in Fleet ({kibana-pull}213486[#213486]).
* Adds the ability for users to manage detection rules externally by utilizing the https://dac-reference.readthedocs.io/en/latest/dac_concept_and_workflows.html[Detection-as-Code principles]. To get started, refer to the https://github.com/elastic/detection-rules/blob/main/README.md#detections-as-code-dac[Elastic detection-rules repository DaC documentation].
```